interface/forms/group_attendance/new.php

```diff
@@ -0,0 +1,105 @@
+<?php
```

Comment:
Add a standard header.

Add the security flags before the below globals.php include:
`````` 
$fake_register_globals=false;
$sanitize_all_escapes=true;
``````

Always use require_once instead of include_once in below includes